nipype/algorithms/modelgen.py

```diff
@@ -144,6 +144,61 @@ def scale_timings(timelist, input_units, output_units, time_repetition):
     timelist = [np.max([0., _scalefactor * t]) for t in timelist]
     return timelist
 
+def bids_gen_info(bids_event_files,
+    condition_column='trial_type',
+    amplitude_column=None,
+    time_repetition=False,
```

Comment:
```suggestion
                  time_repetition=False):
```

Reply:
this is better, because it allows you to add parameters without editing the lines belonging to other parameters.

Reply:
Okay.